oauthenticator/oauth2.py

```diff
@@ -1086,3 +1080,18 @@ def __init__(self, **kwargs):
                 self._deprecated_oauth_trait, names=list(self._deprecated_oauth_aliases)
             )
         super().__init__(**kwargs)
+
+
+# patch allowed_users help string to match our definition
+# base Authenticator class help string gives the wrong impression
+# when combined with other allow options
+OAuthenticator.class_traits()[
+    "allowed_users"
+].help = """
+Set of usernames that should be allowed to login.
+
+If unspecified, grants no access. You must set at least one other `allow` configuration
+if any users are to have permission to access the Hub.
+
+Any users in `admin_users` will be added to this set.
```

Comment:
I made another comment about this, i don't think this is correct.

Reply:
Good call, done.